mrc/control/acting/strategies.py

```diff
@@ -0,0 +1,46 @@
+from mrc.control.acting.abstract import AbstractStrategy
+from mrc.control.movement_prediction.target_position_calculator import TargetPositionCalculator
+from mrc.control.steering.exceptions import ObstacleOnTheWayException, SteeringException
+from mrc.shared.position import PolarPosition
+
+
+class FollowMasterStrategy(AbstractStrategy):
+
+    def __init__(self, steering_strategy, locator, configurator):
```

Comment:
Maybe rename `steering_strategy` to `steering_interface`, it's not really a strategy.

Reply:
It is, but ok.